### PR TITLE
runtime tests commit for pull request

### PR DESCRIPTION
**Pull request by benspector3**
Added tests for Runtime using Cheerio, Sinon, Chai, and Mocha.

The tests follow the outline of the repo README but I think there should be a few changes made to the requirements for completing this project. I have written a document below that outlines how each test was conducted and outlines any changes made to the repo.

[Runtime Testing Document](https://docs.google.com/document/d/1tnWHCHeuSTHype4jkcsKSvoIcHNDRe2TOhrK4ndMXhA/edit?usp=sharing)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "i": "^0.3.6",
+    "lodash": "^4.17.4",
+    "mocha": "^4.0.0",
+    "mock-browser": "^0.92.14",
+    "sinon": "^3.3.0",
+    "zombie": "^5.0.5"
+  },
+  "scripts": {
+    "pretest": "npm install",
+    "test": "mocha ./test/runtime.spec.js"
```
Comment (livrush) [inline package.json]: There should be a another part to the `test` script. You also need the `posttest` script to delete the test directory. Check in the document I sent you a while ago:

https://docs.google.com/document/d/1pDPIsubs3CqtXdo9VilBTE8iH6eiEwZQgsCy5oO0QiE/edit

File: test/mocha.opts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+--use_strict
+-t
```
Comment (livrush) [inline test/mocha.opts]: Should also have `--reporter json` to be compatible with `opspark`

Comment (livrush) [reply test/mocha.opts]: Also, you need an `.npmrc` file

Comment (benspector3) [reply test/mocha.opts]: Just made all changes - Thanks!

**Comment by benspector3** [comment]
@livrush Hey Liv, I've made a number of changes to make the README clearer and to make testing a bit more robust. Let me know what you think!

**Pull request merged**